src/NexusMods.SingleProcess/SyncFile.cs

```diff
@@ -10,53 +10,31 @@ namespace NexusMods.SingleProcess;
 public class SyncFile
 {
     private readonly MultiProcessSharedArray _sharedArray;
+    private readonly ILogger<SyncFile> _logger;
 
     /// <summary>
     /// DI constructor
     /// </summary>
     public SyncFile(ILogger<SyncFile> logger, ISettingsManager settingsManager)
     {
         var settings = settingsManager.Get<CliSettings>();
-
-        _sharedArray = new MultiProcessSharedArray(settings.SyncFile, itemCount: 1);
-    }
-
-    private readonly struct SyncFileContents
-    {
-        public readonly ulong Value;
-
-        public int Port => (int)(Value >> 32);
-
-        public int ProcessId => (int)(Value & 0xFFFFFFFF);
-
-        public SyncFileContents(ulong value)
-        {
-            Value = value;
-        }
-
-        public SyncFileContents(int port, int processId)
-        {
-            var lower = (ulong)port;
-            var upper = (ulong)processId << 32;
-            Value = lower & upper;
```

Comment:
I just realized that this should've been `|` which is why the code wasn't working 😅. Sad to see the struct go, but whatever.